examples/falcon_sensor_download/main.go

```diff
@@ -64,11 +66,21 @@ Falcon Client Secret`)
 				// No version distinction, single package suits all
 				*osVersion = ""
 			} else {
-				fmt.Printf("Missing --os-version command-line option. Available version are: [%s]\n", strings.Join(validOsVersions, ", "))
+				fmt.Printf("Missing --os-version command-line option. Available OS versions are: [%s]\n", strings.Join(validOsVersions, ", "))
 				*osVersion = promptUser("Selected OS Version")
 			}
 		}
-		sensor := querySuitableSensor(client, *osName, *osVersion)
+		if *sensorVersion == "" {
+			validSensorVersions := getValidSensorVersions(client, *osName, *osVersion)
+			if len(validSensorVersions) == 0 {
+				fmt.Fprintf(os.Stderr, "No sensors available for specified os/version tuple: %s/%s\n", *osName, *osVersion)
+				os.Exit(1)
+			}
+			fmt.Printf("Missing --sensor-version=latest command-line option. Available sensor versions are: [%s]\n", strings.Join(validSensorVersions, ", "))
+			*sensorVersion = promptUser("Selected Sensor Version")
```

Comment:
```suggestion
			fmt.Printf("Missing --sensor-version command-line option. Available sensor versions are: [%s]\n", strings.Join(validSensorVersions, ", "))
			*sensorVersion = promptUser("Selected Sensor Version")
```

Reply:
Unfortunately this won't work.

There are multiple compromises that needs to be made here to allow for the least possible amount of surprises. As a result the message in questions (that is rarely to be seen) contains some surprises.